portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -646,4 +754,45 @@ definition {
 		AssertElementNotPresent(locator1 = "Portlet#HEADER_DUPLICATE");
 	}
 
+	@description = "This test check purchase apps in marketplace store and view in purchased."
+	@priority = "3"
+	test ViewPurchasedApp {
```

Comment:
Should this test see the purchased App via Control Panel > MARKETPLACE
 > Purchased, would you please double check, thanks.